aws_lambda_powertools/utilities/parser/models/iot_registry_events.py

```diff
@@ -0,0 +1,110 @@
+import enum
+from typing import Any, Dict, List, Optional
+
+from pydantic import BaseModel
+
+
+class IOTCoreRegistryEventBase(BaseModel):
+    eventType: str
+    eventId: str
+    timestamp: int
+
+
+class IoTCRUDEventOperation(str, enum.Enum):
+    CREATED = "CREATED"
+    UPDATED = "UPDATED"
+    DELETED = "DELETED"
+
+
+class AddRemoveOperation(str, enum.Enum):
+    ADDED = "ADDED"
+    REMOVED = "REMOVED"
+
+
+class IoTCRUDEvent(IOTCoreRegistryEventBase):
+    """The "Thing: created, updated, deleted" event"""
+
+    operation: IoTCRUDEventOperation
+
+    thingId: str
+    thingName: str
+    versionNumber: int
+    thingTypeName: Optional[str]
+    billingGroupName: Optional[str]
+
+    attributes: Dict[str, Any]
```

Comment:
```suggestion
    thingId: str
    accountId: str
    thingName: str
    versionNumber: int
    thingTypeName: Optional[str]
    billingGroupName: Optional[str]

    attributes: Dict[str, Any]
```

accountId is a key in this type of event.